README.md

```diff
@@ -1,5 +1,8 @@
 # Apache Drill
 
+[![Build Status](https://travis-ci.org/apache/incubator-openwhisk.svg?branch=master)](https://travis-ci.org/apache/drill)
```

Comment:
Please fix the link to point to drill (https://api.travis-ci.org/apache/drill.svg?branch=master). Consider adding maven central badge.

Reply:
Oh carp! Thanks for catching this.